contrib/perftest/main.go

```diff
@@ -0,0 +1,261 @@
+	*/
+}
+
+func printSystemInfo(client *libpod.Runtime) error {
+	OCIRuntimeInfo, err := client.GetOCIRuntimeVersion()
+	if err != nil {
+		return err
+	}
+
+	connmanInfo, err := client.GetConmonVersion()
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%s\n%s\n\n", OCIRuntimeInfo, connmanInfo)
+	return nil
+}
+
+func runSingleThreadedStressTest(ctx context.Context, client *libpod.Runtime, imageName string, imageID string, config *cc.CreateConfig, testCount int) (*profileData, error) {
+	data := new(profileData)
+	fmt.Printf("Test Round: ")
+	for i := 0; i < testCount; i++ {
+		fmt.Printf("%d ", i)
+
+		runtimeSpec, err := cc.CreateConfigToOCISpec(config)
+		if err != nil {
+			return nil, err
+		}
+
+		//Create Container
+		networks := make([]string, 0)
+		createStartTime := time.Now()
+		ctr, err := client.NewContainer(ctx,
+			runtimeSpec,
+			libpod.WithRootFSFromImage(imageID, imageName, false),
+			libpod.WithNetNS([]ocicni.PortMapping{}, false, networks),
+		)
+		if err != nil {
+			return nil, err
+		}
+		createTotalTime := time.Now().Sub(createStartTime)
+
+		// Start container
+		startStartTime := time.Now()
+		err = ctr.Start(ctx)
+		if err != nil {
+			return nil, err
+		}
+		startTotalTime := time.Now().Sub(startStartTime)
+
+		//Stop Container
+		stopStartTime := time.Now()
+		err = ctr.StopWithTimeout(2)
+		if err != nil {
+			return nil, err
+		}
+		stopTotalTime := time.Now().Sub(stopStartTime)
+
+		//Delete Container
+		deleteStartTime := time.Now()
+
+		err = client.RemoveContainer(ctx, ctr, true)
+		if err != nil {
+			return nil, err
+		}
+
+		deleteTotalTime := time.Now().Sub(deleteStartTime)
+
+		data.updateProfileData(createTotalTime, startTotalTime, stopTotalTime, deleteTotalTime)
+	}
+	return data, nil
+}
+
+type profileData struct {
+	minCreate, minStart, minStop, minDel time.Duration
+	avgCreate, avgStart, avgStop, avgDel time.Duration
+	maxCreate, maxStart, maxStop, maxDel time.Duration
+}
+
+func (data *profileData) updateProfileData(create, start, stop, delete time.Duration) {
+	if create < data.minCreate || data.minCreate == 0 {
+		data.minCreate = create
+	}
+	if create > data.maxCreate || data.maxCreate == 0 {
+		data.maxCreate = create
+	}
+	if start < data.minStart || data.minStart == 0 {
+		data.minStart = start
+	}
+	if start > data.maxStart || data.maxStart == 0 {
+		data.maxStart = start
+	}
+	if stop < data.minStop || data.minStop == 0 {
+		data.minStop = stop
+	}
+	if stop > data.maxStop || data.maxStop == 0 {
+		data.maxStop = stop
+	}
+	if delete < data.minDel || data.minDel == 0 {
+		data.minDel = delete
+	}
+	if delete > data.maxDel || data.maxDel == 0 {
+		data.maxDel = delete
+	}
+
+	data.avgCreate = data.avgCreate + create
+	data.avgStart = data.avgStart + start
+	data.avgStop = data.avgStop + stop
+	data.avgDel = data.avgDel + delete
+}
+
+func (data *profileData) printProfiledData(testCount float64) {
+
+	fmt.Printf("\nProfile data\n\n")
+	fmt.Printf("\tCreate\tStart\tStop\tDelete\n")
```

Comment:
I recommend using the [tabwriter](https://golang.org/pkg/text/tabwriter/) package instead. It's working here as the digits are limited, but it's harder to extend.